AtrumGeost/events-practice-plugin
# Add PHP documentation to the code and format improvements

**Description:** Adds PHP documentation to the plugin's code and improves the code format (code alignment, missing public keywords inside classes, and spacing)

### Testing instructions

This PR doesn't make changes to the plugin's functionality, it only removes/merges unnecessary files and adds the PHP documentation mentioned above. 

1. Activate the plugin
2. The plugin should display two new menus: Events Practice and My Events.
3. The Event CPT and reservation form that appears with the `[eventspractice_custom_reservation]` shortcode should continue working as before.
